src/Compilers/Core/Portable/Operations/BasicBlock.cs

```diff
@@ -84,28 +84,22 @@ internal void RemovePredecessor(BasicBlock block)
             _predecessors.Remove(block);
         }
 
-        [Flags]
-        public enum BranchFlags
+        public enum BranchKind
         {
+            // Keep 0 value unused to detect an uninitialized state
```

Comment:
Consider adding `None = 0`.

Reply:
> Consider adding None = 0.

I intentionally didn't add this member because API will never expose it to consumers.

---
In reply to: [173242372](https://github.com/dotnet/roslyn/pull/25338#discussion_r173242372) [](ancestors = 173242372)